frontstage/common/authorisation.py

```diff
@@ -33,7 +33,7 @@ def validate(token):
     return True
 
 
-def jwt_authorization(request):
+def jwt_authorization(request, refresh_session=True):
```

Comment:
A bit of inside trading here, I know from past experience that the timeout modal does not want expire-at to update the session when hit